invenio_theme/assets/semantic-ui/less/invenio_theme/theme/modules/checkbox.overrides

```diff
@@ -39,3 +39,50 @@
 .dash:before  { content: '\e801'; }
 .plus:before { content: '\e802'; }
 */
+
+.ui.toggle.checkbox.on-off {
+  position: relative;
+
+  label {
+    position: relative;
+    height: @onOffToggleHeight;
+    padding-left: @onOffToggleWidth;
+
+    &::before {
+      content: "OFF";
+      position: absolute;
+      top: 50%;
+      transform: translateY(-50%);
```

Comment:
isn't translateY(-50%) reverting the top:50% ?

Reply:
No, it's going -50% of the `top: 50%`, so it's centered.